src/diff/children.js

```diff
@@ -18,33 +18,20 @@ import { removeNode } from '../util';
  * which have mounted
  * @param {import('../internal').Component} ancestorComponent The direct parent
  * component to the ones being diffed
+ * @param {Node | Text} childDom The current attached DOM
+ * element any new dom elements should be placed around. Likely `null` on first
+ * render (except when hydrating). Can be a sibling DOM element when diffing
+ * Fragments that have siblings. In most cases, it starts out as `oldChildren[0]._dom`.
  */
-export function diffChildren(parentDom, newParentVNode, oldParentVNode, context, isSvg, excessDomChildren, mounts, ancestorComponent) {
+export function diffChildren(parentDom, newParentVNode, oldParentVNode, context, isSvg, excessDomChildren, mounts, ancestorComponent, childDom) {
```

Comment:
Wondering if `childDom` is better named `oldDom` since it often comes from `oldChildren[i]._dom`...

Reply:
Agree, `oldDom` is a more fitting name. `childDom` implies that it would be the `_dom` property of `childVNode` but that's not the case and always confuses me a bit tbh :+1:

Reply:
Went ahead and renamed it :+1:

Reply:
Thanks!